napari/utils/notifications.py

```diff
@@ -83,12 +83,20 @@ def __init__(
     ):
         self.severity = NotificationSeverity(severity)
         super().__init__(type=str(self.severity).lower(), **kwargs)
-        self.message = message
+        self._message = message
         self.actions = actions
 
         # let's store when the object was created;
         self.date = datetime.now()
 
+    @property
```

Comment:
Can you maybe add a comment here, like "Using a property to the message appears in ...." I could just imagine someone coming back to this code and wondering why message is a property when at first glance it doesn't look like it does anything more